frontends/user-tutorials/sidebar/components/PublishTutorialModal/PublishTutorialModal.tsx

```diff
@@ -50,11 +51,16 @@ export const PublishTutorialModal: FC = () => {
   };
 
   const fileChangeHandler = async (file: File): Promise<void> => {
-    const compressedImage = await compressImage(file);
-    setSelectedImage(compressedImage);
+    try {
```

Comment:
why do we need to wrap it with try catch?

Reply:
@komronbekmuhammadjonov11

Reply:
Faqat success bolganda error  mesage o'chirilsin degandim . try catch foydalanmasa ham togri boladimi @Juratbek

Reply:
@Juratbek  review qilib qaytarilganlarni togrilab push qilingandan song. Yangi pull request ochilmaydimi ?

Reply:
Buni ham togrilab push qilib qoydim